src/smallestPrimeFactor.js

```diff
@@ -11,15 +11,48 @@
  * @param {number} num - A number
  * @returns {number} The smallest prime factor of num
  */
-function smallestPrimeFactor(num) {
+
+  function isPrime(num) {
+    if ( num === 1){
+       return false;}
+
+       if ( num === 2){return true;}
+
+     for ( let n = 2; n < num; n++){
+     if (num % n === 0){
+      return false;
+     }
+     return true;
+   }
+  }
+
+  function smallestPrimeFactors(num) {
+    let array = [];
+    let i = 2; // i-quotient
+    for (i=2; i<= num; i++){
+  if (num % i === 0){
+    if (isPrime(i) === true)
```

Comment:
Since `isPrime` directly returns `true` or `false`, you can do `if(isPrime(i)) { ... }`.